src/models/ServerStyleSheet.js

```diff
@@ -149,36 +151,42 @@ export default class ServerStyleSheet {
     return this.instance.toReactElements()
   }
 
-  interleaveWithNodeStream(readableStream: Readable) {
-    const ourStream = new Readable()
+  interleaveWithNodeStream(readableStream: stream.Readable) {
+    if (__SERVER__) {
```

Comment:
for a few bytes less you could wrap this under the class declaration and assign directly to the prototype
```js
if (__SERVER__) {
  ServerTag.prototype. interleaveWithNodeStream = // ...
}
```

but does this file cas to be included in browser bundles altogether? whats the use case? it also might cause problems for cjs consumers, `stream` package might get tree-shaken easily in es bundles, but in cjs its just another whole story